bundles/ch.elexis.molemax/.classpath

```diff
@@ -1,7 +1,12 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <classpath>
-	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-11"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-11">
+		<attributes>
+			<attribute name="module" value="true"/>
+		</attributes>
+	</classpathentry>
 	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
+	<classpathentry exported="true" kind="lib" path="lib/org.eclipse.nebula.widgets.gallery_1.0.0.202303072132.jar" sourcepath="lib/org.eclipse.nebula.widgets.gallery.source_1.0.0.202303072132.jar"/>
```

Comment:
exportieren solltest du das eigentlich nicht, das ist nur für die verwendung in deinem plugin.
Bei nächsten Target-Update werden wir das ins Target selbst aufnehmen https://redmine.medelexis.ch/issues/23378